test/e2e/networkqos.go

```diff
@@ -0,0 +1,767 @@
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2ekubectl "k8s.io/kubernetes/test/e2e/framework/kubectl"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+)
+
+var _ = ginkgo.Describe("e2e NetworkQoS validation", func() {
+	const (
+		podImage       = "nicolaka/netshoot"
```

Comment:
should be the "fqdn" of the image - docker.io/ or whatever

Reply:
good point! we should actually use the image in github.